### ISSUE TITLE
Forcejoin admin-command

### ISSUE THREAD

**Issue by Ghmmy** [garbagemule/MobArena#659]
# Feature request

**Short description**
Currently we are only able to join arenas through permissions, by adding the forcejoin command, we can forcejoin certain players, parsed through console, after they meet a certain criteria. For example: I set up a menu with a level-50 click-requirement. This would mean, only players above level 50 are able to click the item in the menu. A 2nd example is where players collect tokens and with those tokens they can enter the arena (using the forcejoin [name] command, to bypass the permission). 
With the current /ma join [arena], players are able to join the arena from every area on the world at any time. If I want to add the token idea to one of my arenas, I have to give them the arena-permission, make them join the arena, and then revoke the permission, which could be unstable in some of the cases due to delays. By adding the /forcejoin [player] [arena], we can make the console force players to join an arena, without them requiring the permission. 

Long story short: by implementing /forcejoin [player] [arena], which would have an admin permission bound to it, we can make different requirements to certain arenas other than money and permissions.

**Implementation details**

Adding a `/forcejoin [player] [arena]`, which requires the mobarena.admin permission to be executed. This would mainly be parsed through the console for an extended way of adding different requirements to arenas, as explained above.

**Additional info**

**Comment by garbagemule** [comment]
Duplicate of #594
